pkg/scheduler/factory/factory.go

```diff
@@ -720,11 +720,11 @@ func (c *configFactory) onStorageClassDelete(obj interface{}) {
 			var ok bool
 			sc, ok = t.Obj.(*storagev1.StorageClass)
 			if !ok {
-				klog.Errorf("cannot convert to *storagev1.StorageClass: %v", t.Obj)
+				klog.Errorf("cannot convert DeletedFinalStateUnknown obj to *storagev1.StorageClass: %v", t.Obj)
```

Comment:
If you want to describe the log more clearly, then don't use a shorthand like `obj`.

Reply:
It has printed the obj, I modified it because its description is the same to another log, supplement it for more clear.

Reply:
It has printed the obj, I modified it because its description is the same to another log, supplement it for more clear.